tests/dummy/app/pods/error/template.hbs

```diff
@@ -0,0 +1,47 @@
+<div class='dummy-body error'>
+
+  <div class='section-title'>Error</div>
+  <hr>
+  <div class="section">
+    <div class="example">
+      <div class="demo">
+        {{frost-error
+          hook='myError'
+          description='Service is currently unavailable.'
+          suggestion='Try again later.'
+          errorCode=500
+          errorTitle='Internal server error'
+          errorMessage='Failure to retrieve network elements'
+          errorDetails='Detailed information would be displayed here.'
+          showDetails=showDetails
+        }}
+      </div>
+      <br/>
+      <div class="snippet">
+        Template
+        {{format-markdown "```handlebars
+{{frost-error
+  hook='myError'
+  description='Service is currently unavailable.'
+  suggestion='Try again later.'
+  errorCode=500
+  errorTitle='Internal server error'
+  errorMessage='Failure to retrieve network elements'
+  errorDetails='Detailed information would be displayed here.'
+  showDetails=showDetails
+}}
+```"}}
+        <br/>
+        Controller
+        {{format-markdown "```javascript
+queryParams: {
+  showDetails: {
+    replace: true
+  }
+},
+showDetails: false,
+```"}}
```

Comment:
Can we use `{{code-snippet'}}` instead of this? Then you know the snippet is working, if it comes from the live demo. It also lets the snippet be linted if it exists in a `.js` file.